ilp-core-codecs/src/test/java/org/interledger/core/asn/codecs/InterledgerPreparePacketOerSerializerTests.java

```diff
@@ -74,20 +74,20 @@
         {InterledgerPreparePacket.builder()
             .destination(InterledgerAddress.of("test3.foo.bar"))
```

Comment:
`Fulfillment.of -> InterledgerFulfillment.from`
`Condition.of -> InterledgerCondition.from`

`InterledgerAddress.of -> InterledgerAddress.of` 

?

Reply:
At first glance I think these do different things. `.of` is a constructor from simple input and `.from` is a clone.

Reply:
@adrianhopebailie 
Hrmmm, seems using from for `simple inputs` too: 
e.g.
InterledgerCondition.from(conditionBytes)

Reply:
We did try to have a convention but it's possible things have gotten mixed up with time.

@sappenin WDYT?

Reply:
Yeah, there are some typos in the code. Should be `.of` for construction with inputs, and `.from` for copy-constructor-like behavior. I'll fix the code.

Reply:
Fixed in 8f8b44d9e9bfc2df359a668d75548dbb03b6a0e2 and 552a691651a658126c85f1bc0d14487c04b5a0a2